nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -350,6 +350,12 @@
      See the <literal>fish</literal> <link xlink:href="https://github.com/fish-shell/fish-shell/releases/tag/3.0.0">release notes</link> for more information.
     </para>
    </listitem>
+  <listitem>
+   <para>
+    <literal>types.optionSet</literal> is deprecated. Use <literal>types.submodule</literal>
```

Comment:
I think this needs more strong wording since "deprecated" might imply that it is still actually working. I would suggest something like
```xml
<literal>types.optionSet</literal> has been removed after a long deprecation period. Use <literal>types.submodule</literal> instead.
```
I guess "removed" is not technically correct but I think it is morally correct 😉